### DISCUSSION TITLE
Linux Shell Script (Debian): Challenge (FTP Mount)

### DISCUSSION THREAD

**Discussion by CK1-CK** [acmesh-official/acme.sh#5092] [Q&A]
Hi folks,

I have a question. I want to execute an ACME challenge using a shell script. 

Web host: Hosteurope
FTP Write Access: Yes

In the past, I have mounted the FTP directory using (Fuse) curlftpfs, executed the challenge, created the certificate locally and then imported it manually at the provider.

Unfortunately, FUSE no longer works for me and I need an alternative. 

What is the best solution to get a new certificate quickly and easily? How do you deal with the problem?

1. Mount FTP Directory (in the past curlftpfs)  --> Here is my problem!
2. acme.sh --issue -d example.de --webroot $FTPMOUNT_POINT/ --certpath $cert_directory/server.crt --keypath $cert_directory/server.key
3. umount ftp
4. Manual Upload server.crt, server.key to provider

Many thanks in advance.

Best regards
CK1

**Comment by neilpang** [comment]
use the dns mode or the dns alias mode.
